core/Test/Tasty/ConsoleFormat.hs

```diff
@@ -0,0 +1,24 @@
+module Test.Tasty.ConsoleFormat
```

Comment:
This module was added so that providers can render in the tasty console format without having to re-do some of its logic.